### PR TITLE
ConsoleInteractionTest.py: Updated to reduce redundancy

### LINKED ISSUE #6056: ConsoleInteractionTest.py: `test_ask_for_actions_and_apply` defined twice
Change the name of one of the function to something more appropriate.
difficulty/newcomer

### PR DESCRIPTION
**Pull request by pilgrim2308**
A previous PR was mergeable but needed rebasing. Thanks, @Dilshaad21! :)
Also, made a minor change so LineLengthBear no longer shows an error.

Closes #6056

### CODE REVIEW

File: tests/output/ConsoleInteractionTest.py [hunk 1]
```diff
@@ -553,39 +553,25 @@ def test_ask_for_actions_and_apply(self):
...
+        args = [[self.console_printer, Section(''),
+                 [do_nothing_action.get_metadata(), action.get_metadata()],
+                 {id(do_nothing_action): do_nothing_action, id(action): action},
+                 failed_actions, Result('origin', 'message'), {}, {}, {}],
+                [self.console_printer, Section(''),
+                 [action.get_metadata()], {id(action): action},
+                 failed_actions, Result('origin', 'message'), {}, {}, {}]]
+        for arg in args:
+            with simulate_console_inputs('a', 'param1', 'a', 'param2') as
+            generator:
```
Comment (anshalshukla) [inline tests/output/ConsoleInteractionTest.py:565]: I think this was not necessary, and there are quite a lot of places where an 80 character line length limit is avoided. Splitting it into lines adversely affects code readability. 
Maybe @abhishalya can confirm but as long as fragments in .coafile are passing we don't have to do this.

**Review by anshalshukla — COMMENTED** [review]
I've made a comment. Rest everything looks fine to me.

**Review by akshatkarani — COMMENTED** [review]
Changes look good

**Comment by abhishalya** [comment]
@pilgrim2308 The build fails here, you might want to have a look at the log once.

**Comment by pilgrim2308** [comment]
It's failing because of the change I made to make the code adhere to the line length limit. I don't really agree with the solution of using `#Start ignoring LineLengthBear` because if we make it a uniform practice within the code, we'll ultimately be affecting code readability. How should I proceed? @abhishalya

**Comment by abhishalya** [comment]
You can adhere to line length limit while keeping the syntax correct.

**Comment by pilgrim2308** [comment]
@abhishalya I'm sorry I didn't assess the log correctly before. I think it's ready to merge now. 
I don't understand why the build isn't showing here on Github. It passes [here](https://travis-ci.org/github/pilgrim2308/coala/builds/765020302). PTAL!

**Comment by abhishalya** [comment]
@pilgrim2308 Can you squash commits into one, add yourself as a co-author and try to make the commit message a bit better. I'll get this merged tonight.

**Review by abhishalya — APPROVED** [review]
LGTM :tada:

**Review by anshalshukla — APPROVED** [review]

**Pull request merged**